features/bootstrap/SubjectContext.php

```diff
@@ -554,4 +613,29 @@ public function iShouldSeeTheLatestHighlightedArticlesWithTheMSAInTheList(int $n
             );
         }
     }
+
+    /**
+     * @Then /^I should see (\d+) seniors editors for the MSA \'([^\']*)\' sorted by surname in the 'Senior editors' list$/
+     * @Then /^I should see the (\d+) seniors editors for the MSA \'([^\']*)\' sorted by surname in the 'Senior editors' list$/
+     */
+    public function iShouldSeeSeniorsEditorsForTheMSASortedBySurnameInTheList(int $number, string $subject)
+    {
+        $this->assertSession()->elementsCount('css', '.list-heading:contains("Senior editors") + .listing-list > .listing-list__item', $number + 1);
+
+        $previousEditor = '';
+        for ($i = $number; $i > 0; --$i) {
+            $nthChild = ($number - $i + 1);
+
+            $name = $this->assertSession()->elementExists(
+                'css',
+                '.list-heading:contains("Senior editors") + .listing-list > .listing-list__item:nth-child('.$nthChild.') .profile-snippet__name'
+            );
+
+            if ($name->getText() < $previousEditor) {
```

Comment:
for easier debugging I would get all texts and make a single assertion that the list is ordered